src/js/apps/patients/patient/patient_app.js

```diff
@@ -33,12 +34,17 @@ export default SubRouterApp.extend({
     };
   },
 
-  onBeforeStart() {
+  onBeforeStart({ currentRoute }) {
+    this.actionId = null;
     this.getRegion().startPreloader();
+    this.startRoute(currentRoute);
   },
 
   beforeStart({ patientId }) {
-    return Radio.request('entities', 'fetch:patients:model', patientId);
+    return [
+      Radio.request('entities', 'fetch:patients:model', patientId),
+      this.actionId && Radio.request('entities', 'fetch:actions:model', this.actionId),
```

Comment:
@nmajor25 this is kind of in line with the idea that we don't need to GET the action every time we open a sidebar... _but_ I don't love this pattern because it requires running the `startRoute` in both the `onBeforeStart` and the `onStart`